sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/AlreadyExistException.scala

```diff
@@ -33,6 +33,9 @@ class TableAlreadyExistsException(db: String, table: String)
 class TempTableAlreadyExistsException(table: String)
   extends AnalysisException(s"Temporary table '$table' already exists")
 
+class TempViewAlreadyExistsException(table: String)
+  extends AnalysisException(s"Temporary view '$table' already exists")
```

Comment:
+1 for adding this.

Reply:
We do not want to introduce a new exception type. In contrast, we planned to remove all these exception types because PySpark might output a confusing error message.

Reply:
Oh, I see. Thank you for informing me the direction!

Reply:
ok, I will remove new exception